src/nls/root/strings.js

```diff
@@ -417,6 +417,7 @@ define({
     "CMD_RELEASE_NOTES"                   : "Release Notes",
     "CMD_GET_INVOLVED"                    : "Get Involved",
     "CMD_SHOW_EXTENSIONS_FOLDER"          : "Show Extensions Folder",
+    "CMD_HEALTH_DATA_STATISTICS"          : "HealthData Statistics",
```

Comment:
We should change this to something like "Health Data Sharing."  @ryanstewart, thoughts?

Firefox calls their prefs section "Data Choices," and the menu item to view your data is "Firefox Health Report"... but our menu item combines those two things.  It both shows the data you'll send in the next ping _and_ lets you opt out.